artemis-server/src/main/java/org/apache/activemq/artemis/spi/core/security/ActiveMQJAASSecurityManager.java

```diff
@@ -80,31 +81,41 @@ public ActiveMQJAASSecurityManager(String configurationName, String certificateC
    }
 
    @Override
-   public boolean validateUser(String user, String password) {
```

Comment:
this may break integration with integrated systems, such as wildfly, but it applies to any user maybe?